### PR TITLE
Submit basic functionality for Studio Add Class

### PR DESCRIPTION
**Pull request by rquirbach**
- Page lives under app/studio/[studioID]
- Currently some hard coding for studioID and locationID to be refactored later (see inline comments in code)
- Routes back to app/studio/[studioID] on form submit and click to cancel
- POSTs data to classes table as expected

### CODE REVIEW

File: app/studio/[studioID]/addclass/page.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  //     .select('location_id, name')
+  //     .eq('studio_id', '1');
+  //   if (error) {
+  //     // console.error(error);
+  //   } else {
+  //     // console.log('Fetch Data: ', data);
+  //   }
+  // };
+
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
```
Comment (kahlstorf1) [inline app/studio/[studioID]/addclass/page.tsx]: change event type to handle Input element and Text Area Element to fix type error
e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>

**Review by kahlstorf1 — DISMISSED** [review]
Looks good other than fixing type error I mentioned.

File: app/studio/[studioID]/addclass/page.tsx (hunk 1 above)
Comment (rquirbach) [reply app/studio/[studioID]/addclass/page.tsx]: Thank you for catching this! Change made!

**Review by danny-avila — APPROVED** [review]

**Comment by danny-avila** [comment]
I like your "to-do" comments, well done!

My only feedback is you should style your commit messages without quotations and using semantic commits: https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

**Review by danny-avila — APPROVED** [review]

**Pull request merged**